test/jdk/java/awt/Frame/IMStatusBar.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 1998, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Frame;
+import java.awt.Label;
+import java.awt.Panel;
+import java.awt.TextField;
+
+/*
+ * @test
+ * @bug 4113040
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary Checks that IMStatusBar does not affect Frame layout
+ * @run main/manual/othervm -Duser.language=ja -Duser.country=JP IMStatusBar
+ */
+
+public class IMStatusBar {
+
+    public static void main(String[] args) throws Exception {
+        String INSTRUCTIONS = """
+                If the window appears the right size, but then resizes so that the
+                status field overlaps the bottom label, press Fail; otherwise press Pass.
```

Comment:
does this ALWAYS bring up an IM window ? On all platforms.

Reply:
I checked on all platforms (Win, Linux and Mac), IM window always shown.